graphql/mutation_resolver.go

```diff
@@ -1262,7 +1262,7 @@ func (r *mutationResolver) UpdateBetaFeatures(ctx context.Context, opts UpdateBe
 		return nil, InternalServerError.Send(ctx, fmt.Sprintf("updating beta features: %s", err.Error()))
 	}
 
-	betaFeatures := restModel.APIBetaFeatures{}
+	betaFeatures := evergreen.APIBetaFeatures{}
```

Comment:
Can we add some validation here to ensure users can't enable a beta feature if it isn't enabled globally by the admin flag?

Reply:
+1

Reply:
I think that the validation introduces a number of edge cases that can lead to unexpected behavior. For example, a scenario where:
* Feature A and Feature B exists
* A user already has Feature A enabled on an individual level. Then Feature A is disabled globally.
* The user goes to enable/disable feature B, but when they click save, they receive an error that they cannot save their settings because feature A is enabled.

I'm thinking that instead of validation, I can show the UI like this to avoid confusion:
![active](https://github.com/user-attachments/assets/694cd479-bf6d-4b71-be64-839f37beb9d7)
![inactive](https://github.com/user-attachments/assets/40e5ae5b-425d-438d-bff4-3f883e014f65)

Reply:
Great work so far! It seems like the design has 3 different indicators that represent roughly the same thing -- Active/Inactive badge, Grayed out section, and Enabled/Disabled toggle that all represent the same concept. I think this would be potentially confusing for users. To simplify, I would suggest maybe only keeping the toggle since that's the element users directly interact with. Additionally, to indicate the beta feature is globally disabled, we could consider just hiding it altogether. I think that approach would reduce any confusion into why its not selectable. What do you think about that approach?